src/components/utils/src/file_system.cc

```diff
@@ -63,13 +63,13 @@
 
 uint64_t file_system::GetAvailableDiskSpace(const std::string& path) {
 #ifdef OS_WIN32
-	return 1024 * 1024 * 1024;
+  return 1024 * 1024 * 1024:
```

Comment:
Please change this line code with end of ";". Also please add comment why here DiskSpace should be 3bytes?